react/modules/hideWhenUnavailable.ts

```diff
@@ -0,0 +1,24 @@
+interface Props {
```

Comment:
```suggestion
interface HideWhenUnavailableParams {
```

Reply:
Good catch. I will change it to match the new prop name we are discussing.